bin/pseudo-localize

```diff
@@ -0,0 +1,135 @@
+#! /usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+const getStdin = require('get-stdin');
+
+const { localize } = require('../lib/index');
+
+const argv = require('yargs')
+  .usage(
+    '$0 [src] [options]',
+    'Pseudo localize a string, JSON file, or a JavaScript object',
+    function(yargs) {
+      yargs.positional('src', {
+        describe: 'The source as a path or from STDIN',
+        type: 'string',
+        coerce: path.resolve,
+        conflicts: 'i',
+      });
+    }
+  )
+  .options({
+    o: {
+      alias: 'output',
+      describe:
+        'Writes output to STDOUT by default. Optionally specify a JSON file to write the pseudo-localizations to',
+      type: 'string',
+      coerce: path.resolve,
+    },
+    i: {
+      alias: 'input',
+      describe: 'Pass in direct input to pseudo-localize',
+      type: 'string',
+      conflicts: 'src',
+    },
+    debug: {
+      describe: 'Print out all stack traces and other debug info',
+      type: 'boolean',
+    },
+    pretty: {
+      describe: 'Pretty print JSON output',
+      type: 'boolean',
+    },
+    strategy: {
+      describe: 'Set the strategy for localization',
+      choices: ['accented', 'bidi'],
+      default: 'accented',
+    },
+  })
+  .help()
+  .version().argv;
+
+const debug = function(e) {
+  if (argv.debug) {
+    console.error(e);
+  }
+};
+
+const write = function(output) {
+  argv.output ? fs.writeFileSync(argv.output, output) : fs.writeSync(1, output);
+};
+
+const convert = function(input) {
+  const options = { strategy: argv.strategy };
+
+  if (typeof input === 'string') {
+    write(localize(input, options));
+    return;
+  }
+
+  for (let key in input) {
+    // this will need ot be updated if other primitives
+    // support pseudo-localization
+    if (typeof input[key] === 'string') {
+      input[key] = localize(input[key], options);
+    }
+  }
+
+  write(JSON.stringify(input, null, argv.pretty ? 4 : undefined) + '\n');
+};
+
+(function() {
+  'use strict';
+
+  let json = null;
+
+  if (argv.i) {
+    convert(argv.i + '\n');
+    return;
+  }
+
+  if (argv.src) {
+    try {
+      json = require(argv.src);
+
+      if (json.default) {
+        json = json.default;
+      }
+
+      convert(json);
+    } catch (e) {
+      debug(e);
+
+      console.error(
+        `${e.message}\nUnable to parse input file. Make sure it is in JSON format or is an exported JS module`
+      );
+
+      process.exit(1);
+    }
+  } else {
+    getStdin()
+      .then(function(input) {
+        if (!input) {
+          console.warn('No input from STDIN detected. Exiting...');
+          process.exit(0);
+        }
+
+        let string = input;
```

Comment:
Plain text here is the _default_. We'll still try and parse the input as JSON even if it fails.